packages/gatsby-plugin-now/index.js

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
Why create an index.js file at all?

Reply:
I followed [this guide](https://www.gatsbyjs.org/docs/files-gatsby-looks-for-in-a-plugin/#what-files-does-gatsby-look-for-in-a-plugin) which recommends it